packages/@o3r/create/src/index.ts

```diff
@@ -100,7 +101,8 @@ let defaultPackageManager = 'npm';
 if (packageManagerEnv && ['npm', 'yarn'].includes(packageManagerEnv)) {
   defaultPackageManager = packageManagerEnv;
 }
-const packageManager = argv['package-manager'] || defaultPackageManager;
+const argvPackageManager = argv['package-manager'];
+const packageManager = /(npm|yarn)/.test(argvPackageManager) ? argv['package-manager'] : defaultPackageManager;
```

Comment:
Maybe you want to throw with an error if an invalid (or unsupported) package manager is explicitly given